interfaces/Portal/src/app/components/status-table-filter/status-table-filter.component.ts

```diff
@@ -14,7 +14,7 @@ export class StatusTableFilterComponent {
   public programId: number;
 
   @Input()
-  public thisPhase: ProgramPhase;
+  public thisPhase: ProgramTab;
```

Comment:
I wonder if it is worth it to "refactor the: 'in what phase are some things allowed'-logic totally away" versus "just rename it to something-NOT-called-phase..

Reply:
I think we could live with this term `thisPhase` for now, leave it for later :)